superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/transformProps.ts

```diff
@@ -576,7 +577,12 @@ export default function transformProps(
               ? tooltipFormatter
               : tooltipFormatterSecondary,
           });
-          rows.push(`<span style="opacity: 0.7">${content}</span>`);
+          const { focusedSeries } = ownState;
+          if (key === focusedSeries) {
+            rows.push(`<span style="font-weight: 700">${content}</span>`);
+          } else {
+            rows.push(`<span style="opacity: 0.7">${content}</span>`);
+          }
```

Comment:
not critical but it looks like it always pushes content span but only applies different styles. So this might be better like

```suggestion
          const contentStyle = key === focusedSeries ? 'font-weight: 700' : 'opacity: 0.7';
          rows.push(`<span style="${contentStyle}">${content}</span>`);
```